### ISSUE TITLE
The results about "Cross Entropy" are strange.

### ISSUE THREAD

**Issue by Ethanwl** [MacLLL/SELC#1]
I get the results of `Cross Entropy` by setting `es = 200` in `SELCLoss`.

However, I find that the accuracy is far higher than that in the paper.
For CIFAR-10, when noisy ratio is 0.8, the accuracy achieves 72.06% at epoch 41; when noisy ratio is 0.6, the accuracy achieves 84.33% at epoch 42.

Is there any wrong?

**Comment by MacLLL** [comment]
Hi, set es=200 means training the model with cross entropy loss. The accuracies you mentioned (around epoch 40) require early-stopping. However, as we mentioned in the paper (Section experiments)----we do not perform early stopping since we don’t assume the presence of clean validation data. All test accuracy are recorded from the last epoch of training.

**Comment by Ethanwl** [comment]
Thank you for your reply.
